indy_node/test/upgrade/conftest.py

```diff
@@ -16,46 +19,89 @@ def nodeIds(nodeSet):
     return nodeSet[0].poolManager.nodeIds
 
 
-@pytest.fixture(scope='module')
-def validUpgrade(nodeIds, tconf):
+EXT_PKT_NAME = 'SomeTopLevelPkt'
+
+@pytest.fixture(scope='function', params=[EXT_PKT_NAME, APP_NAME])
+def validUpgrade(nodeIds, tconf, monkeypatch, request):
     schedule = {}
     unow = datetime.utcnow().replace(tzinfo=dateutil.tz.tzutc())
     startAt = unow + timedelta(seconds=100)
     acceptableDiff = tconf.MinSepBetweenNodeUpgrades + 1
     for i in nodeIds:
         schedule[i] = datetime.isoformat(startAt)
         startAt = startAt + timedelta(seconds=acceptableDiff + 3)
-    return dict(name='upgrade-13', version=bumpedVersion(), action=START,
-                schedule=schedule,
-                # sha256=get_valid_code_hash(),
+
+    vers = None
+
+    if request.param != APP_NAME:
+        EXT_PKT_VERSION = '7.88.999'
```

Comment:
It looks like it can be moved to a helper method, as there are other tests with the same mocking.
Also it's better to have it one place when we will support other platforms (not depend on dpkg).

Reply:
done